index.py

```diff
@@ -1 +1,5 @@
-print('hello world')
+import streamlit as st
+
+st.write("# MyApp V3.0")
```

Comment:
```suggestion
st.write("# MyApp")
```

Let's not have version in the title